affinity/cpp-20/d0796r1.md

```diff
@@ -1,29 +1,30 @@
-# D0796r1: Supporting Heterogeneous & Distributed Computing Through Affinity
+# P0796r1: Supporting Heterogeneous & Distributed Computing Through Affinity
 
-**Date: 2017-11-27**
+**Date: 2018-02-12**
 
 **Audience: SG1, LEWG, LWG, SG14**
 
 **Authors: Gordon Brown, Ruyman Reyes, Michael Wong, H. Carter Edwards, Thomas Rodgers, Mark Hoemmen**
 
-**Contributors: Patrice Roy, Jeff Hammond**
+**Contributors: Patrice Roy, Carl Cook, Jeff Hammond**
 
 **Emails: gordon@codeplay.com, ruyman@codeplay.com, michael@codeplay.com, hcedwar@sandia.gov, rodgert@twrodgers.com, mhoemme@sandia.gov**
 
 **Reply to: gordon@codeplay.com**
 
 # Changelog
 
-### P0796R1 (JAX)
+### P0796r1 (JAX)
 
 * Introduce proposed wording.
-* Have `execution_context::resource()` and `this_system::resource()` return a `const execution_resource &`.
+* Based on feedback from SG1, introduced a pair-wise interface for querying the relative affinity between execution resources.
+* Based on feedback from SG1, introduced a non-hierarchical system tpology structure, which doesn't require a root resouce (TODO).
```

Comment:
why is there a TODO?

Reply:
also tpology -> topology

Reply:
The TODO was just a reminder for myself, I've removed it now.

Reply:
Fixed the typo.